### PR TITLE
container: Add function to get container's annotations

### PR DESCRIPTION
**Pull request by devimc**
Annotations returns the container's annotations

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

File: container.go [hunk 1]
```diff
@@ -127,6 +127,11 @@ func (c *Container) URL() string {
 	return c.pod.URL()
 }
 
+// Annotations returns container's annotations
+func (c *Container) Annotations() map[string]string {
+	return c.config.Annotations
```
Comment (sameo) [inline container.go]: Could you elaborate as to why this is needed ?

Comment (devimc) [inline container.go]: https://github.com/clearcontainers/runtime/pull/143

**Comment by devimc** [comment]
@sboeuf  done

File: container_test.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		config: &ContainerConfig{
+			Annotations: annotations,
+		},
+	}
+
+	containerAnnotations := container.GetAnnotations()
+
+	for k, v := range containerAnnotations {
+		if annotations[k] != v {
+			t.Fatal()
```
Comment (sboeuf) [inline container_test.go]: Please add a comment about the failure by using
```go
t.Fatalf("Expecting %q\nGot %q", annotations[k], v)
```

**Comment by devimc** [comment]
@sboeuf still requesting changes ?

**Comment by sboeuf** [comment]
@devimc nope, looks good now. @sameo let us know if this is okay for you.
LGTM

**Comment by jodh-intel** [comment]
The test could be simplified using `reflect.DeepEqual()` but that's a nit.

lgtm.

**Pull request merged**